src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -20,10 +20,15 @@ export CEPH_BUILD_DIR=$CEPH_BUILD_DIR
 source ${MGR_ANSIBLE_VIRTUALENV}/bin/activate
 
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+="py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
+  ENV_LIST+="py3,"
 fi
+ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
 
-tox -c ${TOX_PATH} -e ${ENV_LIST}
+echo "WITH_PYTHON2: ->$WITH_PYTHON2<-"
```

Comment:
can we print out these message only if the test fails?

Reply:
@tchaikov Sure, good idea.